timelock-server/src/integTest/java/com/palantir/atlasdb/timelock/LockWatchValueIntegrationTest.java

```diff
@@ -96,14 +104,22 @@
     private static final TestableTimelockCluster CLUSTER = new TestableTimelockCluster(
             "non-batched timestamp paxos single leader",
             "paxosMultiServer.ftl",
-            generateThreeNodeTimelockCluster(9090, builder -> builder.clientPaxosBuilder(
+            generateThreeNodeTimelockCluster(9096, builder -> builder.clientPaxosBuilder(
```

Comment:
I have a suspicion that 9090 was conflicting (I assume it spins up 9090, 9091, 9092, and we use 9092 elsewhere too).

Reply:
fml